appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt

```diff
@@ -56,13 +56,16 @@ internal class ViewPagerTransformer(
             }
             TransformType.FADE -> {
                 if (position <= -1.0f || position >= 1.0f) {
+                    page.translationX = page.width.toFloat()
```

Comment:
Again here those should not be here.
Here the fix to your problem: https://help.github.com/en/articles/syncing-a-fork